project/appweb/src/main/java/web/TeacherInfo.java

```diff
@@ -0,0 +1,34 @@
+package web;
+
+import bean.Teacher;
+import repository.TeacherRepository;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+@WebServlet(value = "/teacherInfo")
```

Comment:
Некритично для работы, но считается, что в URL должны использоваться символы из нижнего регистра и дефисы.
Поэтому 
```suggestion
@WebServlet(value = "/teacher-info")
```

Reply:
fixed